pose_estimation/include/pose_estimation/pose_estimation.hpp

```diff
@@ -0,0 +1,36 @@
+#include <detector_msgs/centre.h>
+#include <eigen3/Eigen/Core>
+#include <eigen3/Eigen/Geometry>
+#include <nav_msgs/Odometry.h>
+#include <ros/ros.h>
+#include <tf/tf.h>
+
+namespace ariitk::pose_estimation {
+
+class PoseEstimation {
+	public:
+	PoseEstimation();
```

Comment:
Please indent the functions/variables. Clang format doesn't indent after access modifiers.